modules/apps/change-tracking/change-tracking-web/src/main/java/com/liferay/change/tracking/web/internal/portlet/action/UpdatePermissionsMVCActionCommand.java

```diff
@@ -58,23 +58,23 @@ protected void doProcessAction(
 		ThemeDisplay themeDisplay = (ThemeDisplay)actionRequest.getAttribute(
 			WebKeys.THEME_DISPLAY);
 
-		_ctSettingsConfigurationHelper.save(
-			themeDisplay.getCompanyId(),
-			HashMapBuilder.<String, Object>put(
-				"defaultOwnerActionIds",
-				() -> {
-					Role role = _roleLocalService.getRole(
-						themeDisplay.getCompanyId(), RoleConstants.OWNER);
+		try (SafeCloseable safeCloseable =
+				CTCollectionThreadLocal.setProductionModeWithSafeCloseable()) {
 
-					List<String> ownerActionIds = permissions.remove(
-						String.valueOf(role.getRoleId()));
+			_ctSettingsConfigurationHelper.save(
```

Comment:
This safe closable shouldn't be here. Configurations aren't changed track. I think you are trying adjust the ctCollectionId for one of the ConfigurationListeners. Move it there instead.